CMSgov/design-system
# [WNMGDS-1719] theme switcher

## Summary

Adding theme switcher dropdown to the doc site. When the dropdown value changes, the page reloads with the new theme applied.

Additionally, building off Forrest's previous work, if a theme value is saved in session storage, but no theme query param is present, the query param is added to the url (without reloading the page) to keep the two in sync.

## How to test

`yarn start:gatsby` or [demo site](https://cmsgov.github.io/design-system/branch/WNMGDS-1719/theme-switcher/)

You should see the site change if you change the dropdown. You should also see the url query param update as the drop down updates. If you leave the page and come back, the site should maintain your previous theme selection
